src/main/java/org/symphonyoss/symphony/messageml/markdown/nodes/form/TextAreaNode.java

```diff
@@ -0,0 +1,32 @@
+package org.symphonyoss.symphony.messageml.markdown.nodes.form;
+
+/**
+ * Class that Represents a Markdown Node for the "TextArea" form element.
+ *
+ * @author Lucas Macedo
+ * @since 10/21/2019
+ */
+public class TextAreaNode extends FormElementNode {
+  private final static String MARKDOWN = "Text Area";
+
+  private String placeholder;
+  private String initialValue;
+
+  public TextAreaNode(String placeholder, String initialValue) {
+    this.placeholder = placeholder;
+    this.initialValue = initialValue;
+  }
+
+  @Override
+  public String getOpeningDelimiter() {
+    return LEFT_DELIMITER + MARKDOWN;
+  }
+
+  @Override
+  public String getText() {
+    String text = ((placeholder != null) ? "[" + placeholder + "]" : "") +
```

Comment:
Not a big fan of this. String.format would help the readability, but you could instead use a String builder here and have stuff like:

```
StringBuilder markdownRepresentation = new StringBuilder(":");
if(placeholder != null){
   markdownRepresentation.append("[")
          .append(placeholder)
          .append("]");
}
if(initialValue !=null){
 markdownRepresentation.append(initialValue);
}

return markdownRepresentation.build();
```

Reply:
And I know it adds to the number of lines, but I honestly don't believe less lines of code is better as I think the builder example is easier to read.